packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -24,116 +19,126 @@ export const instructionTypeDefs = /* GraphQL */ `
         programId: String
     }
 
+    interface TransactionInstructionJsonParsed {
+        programName: String
+        instructionType: String
```

Comment:
I think that `__type` gives you sufficient information about the type of the instruction, no?